### ISSUE TITLE
Vanadium Redox Power Cell (ZPM) too expensive

### ISSUE THREAD

**Issue by Highbeam** [draknyte1/GTplusplus#385]
# Issue Title  
 I agree the Vanadium Redox was too expensive previously and agree with the increased prices apart from the ZPM ones.

16 Iridium Reinforced Plates are insanely expensive in Iridium and Diamonds.  (64 Iridium, 16 Diamonds plus the extra processing/TNT etc).

I suggest reducing this to 4 or 8.

**Comment by sthegreat** [comment]
I disagree. The cells hold 6.4 billion eu, and upgrading your cells like this nets 4.8 billion eu in additional energy capacity. The equivalent in zpm-tier batteries would be adding about 12 medium naquadah batteries (each costing 16 naquadria, 6 europium, and 2 naquadah alloy cables) or about half of an energy module (one of these costs 16 europium, 4 wetware assemblies, 8 lapotronic energy orb clusters, 2 luv field gens, 128 asoc wafers, 8 smd diodes, and 32 naquadah cables).
